cmd/cmd.js

```diff
@@ -177,17 +178,19 @@ class Cmd {
       .action(function(env, options) {
         checkDeps();
         i18n.setOrDetectLocale(options.locale);
+        const nullify = (v) => (!v || typeof v !== 'string') ? null : v;
         embark.run({
           env: env || 'development',
-          serverPort: options.port,
-          serverHost: options.host,
+          serverPort: nullify(options.port),
+          serverHost: nullify(options.host),
           client: options.client || 'geth',
           locale: options.locale,
-          runWebserver: !options.noserver,
+          runWebserver: options.noserver == null ? null : !options.noserver,
```

Comment:
how come you opted to choose `==` over `===`? Would the option be undefined if not specified?

Reply:
`== null` comparison will (only) match `null` or `undefined`, and I have an old habit of not using bare `undefined` as a value because of pre-ES5 weirdness (undefined could be redefined).